contracts/pool/concentrated/TridentNFT.sol

```diff
@@ -65,7 +65,7 @@ abstract contract TridentNFT {
     /// @param interfaceId XOR of all function selectors in the reference interface.
     /// @return supported Returns 'true' if `interfaceId` is flagged as implemented.
     function supportsInterface(bytes4 interfaceId) external pure returns (bool supported) {
-        supported = interfaceId == 0x80ac58cd || interfaceId == 0x5b5e139f;
+        supported = interfaceId == 0x80ac58cd || interfaceId == 0x5b5e139f || interfaceId == 0x01ffc9a7;
```

Comment:
use `functionName.selector` rather than hardcoding the selectors for better readability.

Reply:
I have made these nits by importing OZ interfaces. This expands code a bit, but adds to suggested legibility.